src/jit/codegen.h

```diff
@@ -465,11 +465,11 @@ protected :
                                     IL_OFFSETX                                    ilOffset);
 
 
+#if defined(_TARGET_ARM_)
     //
```

Comment:
Yes, that is generally the issue. In a lot of cases, I wasn't sure what the comment was commenting on. In some of the cases, the comment really was about what was in the ifdef (like a comment about x86 preceding a #ifdef _TARGET_X86_). I realize that a lot of these changes aren't necessarily accurate. I hadn't thought of your solution, but I could try it.